packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -143,7 +143,7 @@ class DeepLinksController extends DisposableController {
       );
     }
 
-    return _getFilterredLinks(linkDatasByPath.values.toList());
+    return getFilterredLinks(linkDatasByPath.values.toList());
```

Comment:
spelling: getFilteredLinks (this likely will trigger the typo lint for the g3 roll)